src/test/java/org/symphonyoss/symphony/messageml/elements/CodeLanguageTest.java

```diff
@@ -0,0 +1,101 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import static org.junit.Assert.assertThrows;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
```

Comment:
aren't we missing changes in the pom.xml to include junit jupiter?

Reply:
Not since https://github.com/finos/messageml-utils/pull/315

Reply:
Not since https://github.com/finos/messageml-utils/pull/266

Reply:
but is it executed then? I don't see CodeLanguageTest in the PR build for instance

Reply:
Oups 🙊  [196f1fd](https://github.com/finos/messageml-utils/pull/328/commits/196f1fd5665274819c1404022450628f2e2042e9)